### DISCUSSION TITLE
The status bar displays the prompt that "XX files to analyze"

### DISCUSSION THREAD

**Discussion by zhiyiYo** [microsoft/pylance-release#3659] [Enhancement]
## Environment data

-   Language Server version: v2022.11.32
-   OS and version: Windows11
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.6

## Repro Steps
Write some code casually in the text editor

## Expected behavior

Don't show this annoying prompt on the status bar

## Actual behavior
![image](https://user-images.githubusercontent.com/65096819/202837485-51a47890-a966-40ab-9796-f8cf9360b87a.png)

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
